content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -209,7 +121,8 @@ as root:
   kubeadm join --token <token> <master-ip>:<master-port> --discovery-token-ca-cert-hash sha256:<hash>
 ```
 
-To make kubectl work for your non-root user, you might want to run these commands (which is also a part of the `kubeadm init` output):
+To make kubectl work for your non-root user, you might want to run these commands, which are
```

Comment:
s/you might want to//

Maybe "Which are printed at the end of the `kubeadmin init` output. However, since you include the output above, you may not need to reiterate the commands here.  It might be better if the `kubeadm init` output stated why you should run these commands, instead of just telling you to run them with no explanation. It might be even better if kubeadmin supplied a helper script to do these steps. It actually seems like kubeadm should do this for a user the first time that user runs kubeadm commands, instead of requiring a superuser just to copy around config files. And I don't think you need to use sudo in that cp command or the chown command, unless for some reason the file in `/etc/kubernetes` is read-restricted. Typically, default config files are not. As a sysadmin, this whole series of steps feels weird to me. If you're doing it on another user's behalf, you can't use the `$HOME` variable, for instance.